src/System.IO.Compression/src/System/IO/Compression/DeflateZLib/DeflateStream.cs

```diff
@@ -706,6 +706,16 @@ internal ValueTask WriteAsyncMemory(ReadOnlyMemory<byte> buffer, CancellationTok
             }
         }
 
+        public override void CopyTo(Stream destination, int bufferSize)
```

Comment:
We should also update the ref for DeflateStream to include the override.  It's generally not necessary, but not doing so means a type derived from DeflateStream that calls base.CopyTo may end up skipping this implementation, as it may not be a virtual dispatch.

Reply:
:+1: